tests/nested/manual/core20-fault-inject-on-update-config/task.yaml

```diff
@@ -9,8 +9,8 @@ systems: [ubuntu-2*]
 environment:
     TAG/gadget_panic_command_line: update-command-line-gadget
     FAULT/gadget_panic_command_line: panic
-    TAG/gadget_panic_config_bootloader: update-config-bootloader
-    FAULT/gadget_panic_config_bootloader: panic
+    #TAG/gadget_panic_config_bootloader: update-config-bootloader
+    #FAULT/gadget_panic_config_bootloader: panic
```

Comment:
Is this the one that finishes with error? Maybe worth adding a comment on why it is commented out.

Reply:
No, this is an scenario that cannot be reproduced, I'll work on that one in a following pr, I need @valentindavid for that